### DISCUSSION TITLE
A/B Testing

### DISCUSSION THREAD

**Discussion by JustinBack** [tosdr/CrispCMS#11] [Testing]
This is a small progress Tracker about the A/B Testing

**Comment by DrewZane** [comment]
Beta site does not provide feedback on un-indexed URLs/Services.

![image](https://user-images.githubusercontent.com/6953301/100768683-68ca0080-33b8-11eb-84ff-85b55e172906.png)

**Comment by JustinBack** [comment]
I cannot find the service on edit.tosdr.org can you give me a link so I can investigate? The page runs a cron every hour to add missing services

**Comment by DrewZane** [comment]
Disregard - Current UI also does not provide any feedback when user searches un-indexed URLs/Services.

![image](https://user-images.githubusercontent.com/6953301/100769410-3ec50e00-33b9-11eb-8eee-7cfb903ddc02.png)

**Comment by JustinBack** [comment]
Yes, you have to request the service to be added on the forums @ https://forum.tosdr.org/request - We only index added services.

**Comment by DrewZane** [comment]
It may be greatly beneficial to display info about indexing or an empty state result when users search for services which have not yet been indexed.

**Comment by JustinBack** [comment]
Good idea! I get on it soon.

**Comment by JustinBack** [comment]
I have added the empty result info now
![grafik](https://user-images.githubusercontent.com/40178814/100774225-262d1600-3402-11eb-9148-d8f3c936866f.png)
